contracts/satoshi-bridge/src/api/token_receiver.rs

```diff
@@ -8,11 +8,13 @@ pub const GAS_FOR_FT_ON_TRANSFER_CALL_BACK: Gas = Gas::from_tgas(100);
 #[near(serializers = [json])]
 pub enum TokenReceiverMessage {
     DepositProtocolFee,
+    // Here is the withdraw message structure that will be sent from user or dApp to the btc/zcash connector
     Withdraw {
         target_btc_address: String,
         input: Vec<OutPoint>,
         output: Vec<TxOut>,
         max_gas_fee: Option<U128>,
+        orchard_bundle_bytes: Option<String>,
```

Comment:
The naming of this is confusing. It's a String containing the hex bytes of the Orchard section of the transaction. This should probably be documented I think (if it's what you intend).